src/docs/cookbook/persistence/sqlite.md

```diff
@@ -176,20 +185,26 @@ class Dog {
       'age': age,
     };
   }
+
+  // Implement toString to make it easier to see information about
+  // each dog when using the print statement.
+  @override
+  String toString() {
+    return 'Dog{id: $id, name: $name, age: $age}';
+  }
```

Comment:
This was on the full example so I included it also on the extracted code excerpt